tests/e2e/initDevfileTests.sh

```diff
@@ -23,6 +23,16 @@ launchSingleUserstory(){
 checkUserstoryName(){
     local checkedName="$(ls tests/devfiles/${TS_SELENIUM_EDITOR} | grep ${USERSTORY}.spec.ts)";
 
+    if [ -z "${TS_SELENIUM_EDITOR}"]; then
```

Comment:
```suggestion
    if [ -z "$TS_SELENIUM_EDITOR"]; then
```
for the sake of consistency with the script